sources/packages/web/src/components/common/ProgressBar.vue

```diff
@@ -0,0 +1,85 @@
+<template>
```

Comment:
`ProgressBar` seems to be too generic in this case. Should it not be more specific and mention the "steps", for instance? For instance `SteppedProgressBar`.

Reply:
Makes sense. What about StepperProgessBar?

Reply:
That works too.